Gitlab CICD.sublime-syntax

```diff
@@ -0,0 +1,32 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/3/syntax.html
+scope: source.yaml.pipeline.gitlab
+version: 2
+
+extends: Packages/YamlPipelines/YamlPipeline.sublime-syntax
+
+file_extensions:
+  - .gitlab-ci.yml
+
+contexts:
+  node:
+    - meta_prepend: true
+    - match: ^\s+(script)\s*(:)(?=\s|$)
```

Comment:
Please also add highlighting for [`before_script`](https://docs.gitlab.com/ee/ci/yaml/#before_script) and [`after_script`](https://docs.gitlab.com/ee/ci/yaml/#after_script).

Reply:
done in https://github.com/SublimeText/YamlPipelines/commit/0968d9b16bc8d06ee12f6c040d81a0490186a4cf